File: RecyclerViewExample/app/src/main/java/com/celalkorucu/recyclerviewexample/MainActivity.java
package com.celalkorucu.recyclerviewexample;

import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.LinearLayoutManager;

import android.app.Activity;
import android.os.Bundle;
import android.renderscript.ScriptGroup;
import android.view.View;

import com.celalkorucu.recyclerviewexample.databinding.ActivityMainBinding;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {

ActivityMainBinding binding ;
    ArrayList<String> list ;
RCAdapter adapter;
    @Override
    protected void onCreate(Bundle savedInstanceState) {


        super.onCreate(savedInstanceState);
        binding = ActivityMainBinding.inflate(getLayoutInflater());
        View view = binding.getRoot();
        setContentView(view);
        list = new ArrayList<>();


        list.add("Celal KORUCU");
        list.add("İsmail KORUCU");
        list.add("Havva KORUCU");
        list.add("Halime Nur KORUCU");
        list.add("Fatma Sena KORUCU");
        list.add("Mustafa KORUCU");
        list.add("Serhat YÖRÜK");
        list.add("Meryem DEMİRÖRS");
        list.add("Demirci MEHMET");
        list.add("Zeynep Sude KEÇECİ");
        list.add("Melisa ÇAKICI");

        binding.recyclerView.setLayoutManager(new LinearLayoutManager(this));
        adapter = new RCAdapter(list);


binding.recyclerView.setAdapter(adapter);







    }


}

File: KotlinLandmarkBook/app/src/main/java/com/celalkorucu/kotlinlandmarkbook/MainActivity.kt
package com.celalkorucu.kotlinlandmarkbook

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import androidx.recyclerview.widget.LinearLayoutManager
import com.celalkorucu.kotlinlandmarkbook.databinding.ActivityMainBinding

var arrayList = ArrayList<Landmark>()

lateinit var binding: ActivityMainBinding
class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        var view : View = binding.root
        setContentView(view)


        var eiffel = Landmark("Eiffel" , "France" , R.drawable.eiffel)
        var ayasofya = Landmark("Ayasofya" , "İstanbul" , R.drawable.ayasofya)
        var anitkabir = Landmark("Anıtkabir" , "Ankara" , R.drawable.anitkabir)

        arrayList.add(eiffel)
        arrayList.add(ayasofya)
        arrayList.add(anitkabir)



        binding.recyclerView.layoutManager = LinearLayoutManager(this)
        var adapter = LandmarkAdapter(arrayList)
        binding.recyclerView.adapter = adapter




    }
}